### PR TITLE
fix engram tagging

### PR DESCRIPTION
**Pull request by delphiactual**
Addresses #1342

### CODE REVIEW

File: src/scripts/move-popup/dimMoveItemProperties.directive.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ function MoveItemProperties() {
       '      </div>',
       '      <div class="item-type-info">{{vm.light}} {{::vm.classType}} {{::vm.item.typeName}}</div>',
       '      <div ng-if="vm.item.objectives" translate-values="{ percent: vm.item.percentComplete }" translate="ItemService.PercentComplete"></div>',
-      '      <dim-item-tag ng-if="vm.item.lockable && vm.featureFlags.tagsEnabled" item="vm.item"></dim-item-tag>',
+      '      <dim-item-tag ng-if="vm.featureFlags.tagsEnabled && vm.item.lockable && !vm.item.isEngram" item="vm.item"></dim-item-tag>',
```
Comment (bhollis) [inline src/scripts/move-popup/dimMoveItemProperties.directive.js]: `isEngram` is a function, not a property. The reason I haven't done this yet is that I don't want to call a function in an expression (it's bad for perf) - we'd need to write it into each item when they're built.

**Comment by delphiactual** [comment]
@bhollis requested change has been addressed

File: src/scripts/services/dimStoreService.factory.js [hunk 2]
```diff
@@ -743,6 +743,7 @@ function StoreService(
       visible: true,
       sourceHashes: itemDef.sourceHashes,
       lockable: normalBucket.type !== 'Class' && ((currentBucket.inPostmaster && item.isEquipment) || currentBucket.inWeapons || item.lockable),
+      taggable: item.lockable && !item.isEngram,
```
Comment (bhollis) [inline src/scripts/services/dimStoreService.factory.js]: isEngram isn't a property of item, is it?

Comment (delphiactual) [inline src/scripts/services/dimStoreService.factory.js]: its a function of it

Comment (bhollis) [inline src/scripts/services/dimStoreService.factory.js]: It's a function of the item you're building `createdItem` via `ItemProto` - not the raw Bungie item data. And even if it were, you'd have to call that function, no?

**Pull request merged**